tools/mypy/requirements.txt

```diff
@@ -0,0 +1,2 @@
+mypy ~= 1.15.0
```

Comment:
is this overconstrained?

Reply:
https://peps.python.org/pep-0440/#compatible-release shouldn't be, this will match anything in the 1.15 line.